src/style.css

```diff
@@ -0,0 +1,115 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: aliceblue;
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  
+  --card-color: #00acdc;
+  --basic-width: 170px;
+}
+
+.card {
+  width: 200px;
+  padding: 29px 0 15px;
```

Comment:
set the left and right padding to 14 pixels so as not to set the basic-width for the elements